ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/KsqlRequest.java

```diff
@@ -39,18 +39,20 @@
   private final String ksql;
   private final ImmutableMap<String, Object> streamsProperties;
   private final Optional<Long> commandSequenceNumber;
+  private final Optional<Boolean> isInternalRequest;
 
   @JsonCreator
   public KsqlRequest(
       @JsonProperty("ksql") final String ksql,
       @JsonProperty("streamsProperties") final Map<String, ?> streamsProperties,
-      @JsonProperty("commandSequenceNumber") final Long commandSequenceNumber
-  ) {
+      @JsonProperty("commandSequenceNumber") final Long commandSequenceNumber,
+      @JsonProperty("isInternalRequest") final Boolean isInternalRequest) {
     this.ksql = ksql == null ? "" : ksql;
     this.streamsProperties = streamsProperties == null
         ? ImmutableMap.of()
         : ImmutableMap.copyOf(serializeClassValues(streamsProperties));
     this.commandSequenceNumber = Optional.ofNullable(commandSequenceNumber);
+    this.isInternalRequest = Optional.ofNullable(isInternalRequest);
```

Comment:
Rather than an optional, can't we just default this to `false`?

Also, it's normal to _not_ prefix boolean fields with `is`.

```suggestion
    this.internalRequest = internalRequest == null ? false : internalRequest;
```